packages/client/utils/isTaskArchived.ts

```diff
@@ -0,0 +1 @@
+export default (tags) => !!tags.includes('archived')
```

Comment:
`tags: string[]`
...since this is already pretty old & there's not any learnings to be had in this PR i'm just gonna make the changes in a commit to save time